packages/lib-sourcify/src/lib/CheckedContract.ts

```diff
@@ -898,17 +906,22 @@ function findAuxdataPositions(
     }
     // New diff position
     for (const auxdataDiffIndex in auxdataDiffObjects) {
+      const auxdataPositionsIndex = parseInt(auxdataDiffIndex) + 1;
       if (
-        auxdataPositions[auxdataDiffIndex] === undefined &&
+        auxdataPositions[auxdataPositionsIndex] === undefined &&
         bytecodeIncludesAuxdataDiffAt(
           originalBytecode,
           auxdataDiffObjects[auxdataDiffIndex],
           diffPosition
         )
       ) {
-        auxdataPositions[auxdataDiffIndex] = {
-          offset: diffPosition - auxdataDiffObjects[auxdataDiffIndex].diffStart,
-          value: auxdataDiffObjects[auxdataDiffIndex].real,
+        auxdataPositions[auxdataPositionsIndex] = {
+          offset:
+            (diffPosition -
+              auxdataDiffObjects[auxdataDiffIndex].diffStart -
+              2) /
+            2,
+          value: `0x${auxdataDiffObjects[auxdataDiffIndex].real}`,
```

Comment:
Was this also wrong?

Reply:
Yes, this was wrong but it wasn't actually used because we didn't support contracts with multiple auxdata